pkgs/development/python-modules/rotary-embedding-torch/default.nix

```diff
@@ -0,0 +1,21 @@
+{ python3Packages, lib }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "rotary-embedding-torch";
+  version = "0.1.5";
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-OgUvFk6fQtbbxZMWHZA6MwA8uRkP3GC3GbO7MflcjPY=";
+  };
+  propagatedBuildInputs = [
+    python3Packages.pytorch-bin
+    python3Packages.einops
```

Comment:
Please import these dependencies at the top, removing the `python3Packages.` prefix.

Reply:
> I assume this package bundles the weights of a pretrained model. Those are often a couple of megabytes. What is our policy on including that in nixpkgs?

Last time I tested, this tool downloaded the compiled models on the first run